### ISSUE TITLE
[Feature Request] Pre-trained RetinaNetLite-MobileNetV2 model

### ISSUE THREAD

**Issue by netanel-s** [tensorflow/models#5632]
### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.10.
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: CUDA 9
- **GPU model and memory**: 1080Ti, 12GB
- **Exact command to reproduce**: N/A
------------------------

### Describe the problem
About a month ago you added support for FPN on top of MobileNetV2.
Could you please provide a pre-trained model of RetinaNetLite-MobileNetV2?
i.e. depth-wise separable FPN + depth-wise separable detection head comprised of two decoupled 4-layer detection towers + focal loss.

Thanks a lot in advance.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
